package.json

```diff
@@ -143,5 +143,8 @@
     "hooks": {
       "pre-commit": "lint-staged"
     }
+  },
+  "dependencies": {
+    "p5": "file:"
```

Comment:
I think we don't need these changes just to change the documentation. You can reset them with `git checkout main -- package-lock.json package.json` and then commit the results.